css/styles.css

```diff
@@ -145,7 +145,7 @@ cursor: pointer;
 /* about */
 .about{
   height:100%;
-  margin-left: 20%;
+  margin-left:300px;
```

Comment:
Great, Can you set padding in order to put content in middle? I will look better if that line is not shown there.
![Screenshot from 2020-10-21 08-25-45](https://user-images.githubusercontent.com/39116572/96681170-150ad700-1377-11eb-8921-33ea49a3249a.png)